.github/workflows/build.yml

```diff
@@ -21,35 +21,31 @@ on:
           - ztl
           - all
       recipe_file:
-        description: "Filename of recipe to use"
+        description: "Recipe filename to use"
         type: choice
         options:
           - recipe
           - recipe-minor
         required: true
         default: recipe
       version:
-        description: "Version of dataset being built"
+        description: "Version of dataset being built (overrides version defined in recipe)"
```

Comment:
A note - there's no override for socrata sources, for those it always looks up "rowsUpdatedAt". Do you think we should have that manual override be possible?

Reply:
your talkin about setting a version when archiving source data? this is just the version of the product build so I'm not sure this action could/should do anything to versions of sources

I think it'd be nice to have the option to override. rowsUpdatedAt is a great default for socrata sources. but if data changes >once a day and we don't wanna overwrite an archive or start using timestamps in versions, an override in data_library_single.yml would be nice

Reply:
Oh oops. That is what I was referring to